mh/constants.py

```diff
@@ -164,13 +165,41 @@ def make_binary_npc_greeters(is_jap=False):
     US_OFFSET = 0x180
     JP_OFFSET = 0x100
     offset = JP_OFFSET if is_jap else US_OFFSET
+
+
+    if is_jhen_active():
+        tool_shop_str = "Half-off sale!"
+        material_shop_str = "Half-off sale!"
+        event_quests_str = "The Festival of Fear\n"
+    else:
+        tool_shop_str = "Prices are normal."
+        material_shop_str = "Prices are normal."
+        event_quests_str = ""
```

Comment:
Use bytes literals (`b""` so you don't have to encode the strings).

Reply:
Done, except for event quests due to the `get_quest_name` returning a normal string

Reply:
If my comment on `get_quest_name` is addressed, that's doable.

Reply:
Done